integration_tests/pd_api_test.go

```diff
@@ -103,42 +105,107 @@ func (c *storeSafeTsMockClient) CloseAddr(addr string) error {
 	return c.Client.CloseAddr(addr)
 }
 
-func (s *apiTestSuite) TestGetStoreMinResolvedTS() {
+func (s *apiTestSuite) TestGetStoresMinResolvedTS() {
 	util.EnableFailpoints()
-	// Try to get the minimum resolved timestamp of the store from PD.
 	require := s.Require()
+	require.Nil(failpoint.Enable("tikvclient/mockFastSafeTSUpdater", `return()`))
```

Comment:
```suggestion
	require.NoError(failpoint.Enable("tikvclient/mockFastSafeTSUpdater", `return()`))
```

Reply:
seems that this usage in `client-go` needs to be unified